operations/customer-success/README.md

```diff
@@ -129,3 +132,31 @@ Below are detailed steps on how to log a Customer Reference in Salesforce.
    - **Approved by Mattermost Legal:** Owner of the reference has reached out to Mattermost legal to ensure there are no clauses in the MSA that the mattermost cannot use their logo or publish case studies.
```

Comment:
```suggestion
   - **Approved by Mattermost Legal:** Owner of the reference has reached out to Mattermost legal to ensure there are no clauses in the MSA that Mattermost cannot use their logo or publish case studies.
```

Reply:
@justinegeffen I'm good with this change.